src/Microsoft.DotNet.VersionTools/Automation/VersionsRepoUpdater.cs

```diff
@@ -188,6 +197,12 @@ public VersionsRepoUpdater(GitHubAuth gitHubAuth, GitHubProject project)
                 path,
                 new GitHubBranch("master", _project));
 
+            if (latestPackages == null)
+            {
+                Trace.TraceInformation($"No versions repository file '{path}' found.");
```

Comment:
What do you think about combining this trace statement with the one on lines 119-121?